mbedtls/Cargo.toml

```diff
@@ -38,9 +38,13 @@ chrono = "0.4"
 [dependencies.mbedtls-sys-auto]
 version = "2.25.0"
 default-features = false
-features = ["custom_printf", "trusted_cert_callback", "threading"]
+features = ["trusted_cert_callback", "threading"]
 path = "../mbedtls-sys"
 
+[dependencies.mbedtls-selftest]
+version = "0.1"
+path = "../mbedtls-selftest"
```

Comment:
Don't add it as a path dependency for the older branches

Reply:
Removed